src/stash/src/sqlite.rs

```diff
@@ -39,8 +39,7 @@ CREATE TABLE data (
     key blob NOT NULL,
     value blob NOT NULL,
     time integer NOT NULL,
-    diff integer NOT NULL,
-    UNIQUE (collection_id, key, value, time)
+    diff integer NOT NULL
```

Comment:
Such changes to the schema only work without explicit versioning because this is the first release this SQL file / table will be present, right ?

Reply:
that's right